pkgs/top-level/all-packages.nix

```diff
@@ -26275,7 +26275,28 @@ with pkgs;
   linkerd_edge = callPackage ../applications/networking/cluster/linkerd/edge.nix { };
   linkerd_stable = linkerd;
 
-  kumactl = callPackage ../applications/networking/cluster/kumactl { };
+  kuma = callPackage ../applications/networking/cluster/kuma { isFull = true; };
+  kuma_experimental = callPackage ../applications/networking/cluster/kuma {
+    isFull = true;
+    enableGateway = true;
+    pname = "kuma_experimental";
+  };
```

Comment:
Nit: why use underscores in this case?  Elsewhere you put dashes and those [are preferred also generally](https://nixos.org/manual/nixpkgs/stable/#sec-package-naming).

Reply:
many thanks for review @vcunat 
I have missed it. 
I just change it now